jwql/jwql_monitors/monitor_filesystem.py

```diff
@@ -165,7 +238,10 @@ def initialize_results_dicts():
     instrument_results_dict = {}
     instrument_results_dict['date'] = now
 
-    return general_results_dict, instrument_results_dict
+    central_storage_dict = {}
+    central_storage_dict['date'] = now
```

Comment:
@cracraft I think this is the reason that the code is crashing when trying to update the database -- because the database table for the `central_storage` table employs a `unique=True` condition on the `date` column, we won't be able to add multiple records when the `date` is the same for each record, as it is initialized as such here.

I think you could solve this by simply removing the `unique` constraint on the `date` column.  I think this will be fine since we will be running this script via a cron job that executes once a day.

Reply:
FYI, here is the error being thrown:

```
06/03/2019 15:10:23 PM CRITICAL: Traceback (most recent call last):
  File "/Users/bourque/anaconda2/envs/jwql/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1244, in _execute_context
    cursor, statement, parameters, context
  File "/Users/bourque/anaconda2/envs/jwql/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
psycopg2.IntegrityError: duplicate key value violates unique constraint "central_storage_date_key"
DETAIL:  Key (date)=(2019-06-03 11:32:43.48001) already exists.
```

Reply:
I have removed the unique constraint and tested it again and still get the same error. I'm not sure what's going on with this.

Reply:
@cracraft I can try testing this out and digging into it.  I'll report back.

Reply:
@cracraft It took 15 hours to run, but it appears my latest changes (https://github.com/spacetelescope/jwql/pull/401/commits/e7355147d5eff29f7f35cdd3207fd8564b7d543e) results in a successful completion of the filesystem monitor.  Could you try pulling these changes and testing on your end?  If it works on your end, this PR should be ready to go.

Reply:
I didn't pull your edits over before getting back to my testing, but pretty much have the same edits in place and have the code running. I've added more logging so we can see how long it's taking to search each directory and see where the time sink is in this. I've also started adding some code to reduce the duplicate counting, so I'm not ready to commit it just yet.